src/views/BlogPostList/BlogPostList.vue

```diff
@@ -94,23 +92,27 @@ export default defineComponent({
 
     /** Fetch Post list */
 
-    const [
-      fetchPostList,
-      { data: postList, loading: isPostLoading, error },
-    ] = useResource<Array<PostShort>>({
-      fetchResource: getBlogPostList,
+    const {
+      fetchEntityList,
+      isLoading: isPostLoading,
+      rowData,
+      errorMessage,
+      searchQuery,
+      handleChange,
+    } = useDataTable<PostShort>({
+      fetchEntityList: getBlogPostList,
```

Comment:
it should look like
`fetchEntityList: (params) => getBlogPostList({ query: params.searchQuery })`